pkg/tests/ossm-federation/testdata/failover/west-mesh/smcp.yaml

```diff
@@ -1,25 +1,24 @@
 apiVersion: maistra.io/v2
 kind: ServiceMeshControlPlane
 metadata:
-  name: fed-export
+  name: west-mesh
```

Comment:
Same here, we need to add ThirdParty to security section because is going to be executed in ROSA

Reply:
Ignore this, single cluster needs the variable in ROSA is not needed